website/static/js/signIn.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Sign in view model used for login components
+ */
+'use strict';
+
+var ko = require('knockout');
+require('knockout-validation').init({insertMessages: false});  // override default DOM insertions
+var $ = require('jquery');
+
+var $osf = require('osfHelpers');
+
+var ViewModel = function() {
+
+    var self = this;
+
+    self.username = ko.observable('').extend({
+        required: true,
+        email: true
+    });
+    self.password = ko.observable('').extend({
+        required: true,
+        minLength: 6,
```

Comment:
I pulled this min/max from the behavior on /login/ but didn't see these documented anywhere. @sloria thoughts?